setup.py

```diff
@@ -22,7 +22,7 @@
         ],
     },
     install_requires=[
-        'rocketchat_API==0.6.21',
+        'rocketchat_API==0.6.34',
```

Comment:
This probably doesn't work with the rocketchat version installed on the servers. Could you test all the requests that the plugin makes ?

Reply:
It does not matter, we must upgrade the version of the plugin and install it in the Pearson instances, the current installations will work with the previous version.

Reply:
if you can not ensure that it will works with the previous version, please use the version 0.6.21. for the following reasons
1) you have not test all the api calls so there is a possibility that some methods need to be changed.
2) There's no ticket to update the pearson rocketchat server, so probably they wants to use the same one.
3) Maybe in the future they will want to change or add something to rocketchat in the shared instance.

Reply:
First, I've tested the plugin in the Ironwood instance and it works, shared platform is Ginkgo, we are upgrading to Ironwood, if they need anything for the old version instance (Ginkgo) we must create another branch for that and making the changes there and move forward with master.
We can't handle the same xblock version for two different platform version you know that. The version that works for Ginkgo is the previous one, the version that will work for Ironwood is this one.
Every time we can upgrade to a patch version we must update since those changes includes patch updates.

Reply:
Checked.